dev-server/src/server.ts

```diff
@@ -23,6 +24,7 @@ export const runDevServer = (
   const server = createServer(app);
 
   app.use(express.json());
+  app.use(cors())
```

Comment:
with this lines 30:32 can be removed
wonder which header does cors() add that 30:32 doesn't